.github/workflows/create-new-tag-version-label.yml

```diff
@@ -1,16 +0,0 @@
-name: Create new tag as version label
```

Comment:
deleting of this yml file is intentional?

Reply:
Yes, it was deleted as part of 63ec8efe58b4a4d768ef065239acbb9c5c67b6fc.